doc/sphinx/practical-tools/coq-commands.rst

```diff
@@ -219,6 +219,59 @@ and ``coqtop``, unless stated otherwise:
 :-batch: Exit just after argument parsing. Available for ``coqtop`` only.
 :-verbose: Output the content of the input file as it is compiled.
   This option is available for ``coqc`` only.
+:-native-compiler (yes|no|ondemand): Enable the :tacn:`native_compute`
+  reduction machine and precompilation to ``.cmxs`` files for future uses
+  by :tacn:`native_compute`.
+  ``yes`` and ``no`` enable/disable both :tacn:`native_compute` (any call
+  to it will default back to :tacn:`vm_compute`) and precompilation.
+  Note that with ``yes``, compilation will fail if any dependency has not
+  been precompiled itself. ``ondemand`` enables :tacn:`native_compute`
+  but disables precompilation. All missing dependencies will then temporarily
+  be recompiled on the fly when caling :tacn:`native_compute`, which can be
+  very slow. Since version 8.13, the default value is set at configure time,
```

Comment:
Up to you, but you could use the `.. versionchanged::` Sphinx directive here. Its usage is the same as for `.. deprecated::`.